jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/skaffold/SyncMapMojoTest.java

```diff
@@ -0,0 +1,106 @@
+package com.google.cloud.tools.jib.maven.skaffold;
```

Comment:
copyright

Reply:
actually this file shouldn't be in here, git is acting all crazy on my computer